common/config/betterstack.go

```diff
@@ -0,0 +1,87 @@
+package config
+
+import (
+	"errors"
+
+	"github.com/odigos-io/odigos/common"
+)
+
+const (
+	BetterStackMetricsEndpoint = "BETTERSTACK_METRICS_ENDPOINT"
+	BetterStackLogsEndpoint    = "BETTERSTACK_LOGS_ENDPOINT"
+)
+
+var (
+	ErrorBetterStackMetricsEndpointMissing = errors.New("BetterStack is missing a required field (\"BETTERSTACK_METRICS_ENDPOINT\"), BetterStack will not be configured")
+	ErrorBetterStackLogsEndpointMissing    = errors.New("BetterStack is missing a required field (\"BETTERSTACK_LOGS_ENDPOINT\"), BetterStack will not be configured")
+)
+
+type BetterStack struct{}
+
+func (j *BetterStack) DestType() common.DestinationType {
+	return common.BetterStackDestinationType
+}
+
+func (j *BetterStack) ModifyConfig(dest ExporterConfigurer, cfg *Config) error {
+	config := dest.GetConfig()
+	uniqueUri := "betterstack-" + dest.GetID()
+
+	processorName := "attributes/betterstack"
```

Comment:
If someone adds 2 instances of this destination, we will attempt to add this processor twice, and then might get errors or issues. Please add the `dest.GetID()` to the name of the components configured by this destination